Dockerfile

```diff
@@ -74,7 +74,8 @@ RUN cget -p $PREFIX install facebook/zstd@v1.4.5 -X subdir -DCMAKE_DIR=build/cma
 RUN cget -p $PREFIX install ccache@v4.1 -DENABLE_TESTING=OFF
 
 # Install newer cmake for onnx runtime
-RUN cget -p /opt/cmake install kitware/cmake@v3.13.4
+ARG CMAKE_VERSION=3.24.2
+RUN cget -p /opt/rocm install -X binary https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSION}/cmake-${CMAKE_VERSION}-Linux-x86_64.tar.gz
 
```

Comment:
This isn't right.  I tried this and got a crazy install path... 

/opt/rocm/cget/pkg/_url_Z2l0aHViLmNvbS9LaXR3YXJlL0NNYWtlL3JlbGVhc2VzL2Rvd25sb2FkL3YzLjI0LjIvY21ha2UtMy4yNC4yLUxpbnV4LXg4Nl82NC50YXIuZ3o_/install/bin/

Reply:
Werid I just rebuilt a container with this.

Reply:
It should be `-p /opt/cmake`, not `-p /opt/rocm`.